ibis/backends/pandas/client.py

```diff
@@ -120,15 +121,14 @@ def from_pandas_categorical(value):
     return dt.Category()
 
 
-@dt.infer.register(
-    (np.generic,)
-    + tuple(
-        frozenset(
-            np.signedinteger.__subclasses__()
-            + np.unsignedinteger.__subclasses__()  # np.int64, np.uint64, etc.
-        )
-    )  # we need this because in Python 2 int is a parent of np.integer
-)
+_numpy_scalar_types: Tuple[Type, ...] = (np.generic,)
+# we need this because in Python 2 int is a parent of np.integer
```

Comment:
We do not support Python 2 anymore, maybe this is not needed anymore.

Reply:
Done! I've keep only `np.generic` and it is enough to catch all numpy dtypes.